### PR TITLE
Add tests to verify getting flows going through a particular link

### PR DESCRIPTION
**Pull request by ylobankov**
Part of #1370

### CODE REVIEW

**Comment by rtretyak** [comment]
merge it to #1346? Test won't pass without #1386 code

**Comment by ylobankov** [comment]
I will do it.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/links/LinkSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+        and: "Delete all created flows"
+        [flow1, flow2, flow3, flow4].every { northboundService.deleteFlow(it.id) }
+    }
+
+    def "Get flows for NOT existing link"() {
+        when: "Get flows for NOT existing link"
+        def isl = topology.islsForActiveSwitches.first()
+        def response = northboundService.getLinkFlows(isl.srcSwitch.dpId, 1000, isl.dstSwitch.dpId, 1001)
+        println(response)
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/links/LinkSpec.groovy]: remove

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/links/LinkSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        flow4.maximumBandwidth = 1000
+        flow4 = northboundService.addFlow(flow4)
+        assert Wrappers.wait(5) { northboundService.getFlowStatus(flow4.id).status == FlowState.UP }
+
+        when: "Get all flows going through the link from source switch to 'internal' switch"
+        def isl = pathHelper.getInvolvedIsls(PathHelper.convert(northboundService.getFlowPath(flow3.id))).first()
+        def linkFlows = northboundService.getLinkFlows(isl.srcSwitch.dpId, isl.srcPort, isl.dstSwitch.dpId, isl.dstPort)
+
+        then: "All created flows are in response list"
+        [flow1, flow2, flow3, flow4].every { it in linkFlows }
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/links/LinkSpec.groovy]: check result for flow1/2 unique link, it should not contain flow3/4

Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/links/LinkSpec.groovy]: Done.

**Pull request merged**